packages/docusaurus-theme-classic/src/theme/DocPage/styles.module.css

```diff
@@ -67,6 +67,8 @@
 
   .expandSidebarButtonIcon {
     transform: rotate(0);
+    /* todo: use variable */
+    color: #7a7a7a;
```

Comment:
previously color of this icon was hardcoded into component, and there was no way to change it without swizzling, 

![image](https://user-images.githubusercontent.com/625469/144706753-8e1b1501-e2fb-446f-90e6-8bcfb75f51d6.png) ![image](https://user-images.githubusercontent.com/625469/144706765-6ee2ebde-f2df-425d-9fa9-18a73e7194b5.png)

Reply:
any thoughts about this? we could actually remove this override and let control font color control how this icon looks

Reply:
i created issue for this in infima https://github.com/facebookincubator/infima/issues/193